### PR TITLE
docs: Adding a minimal Health Probes Tutorial

### PR DESCRIPTION
**Pull request by akshaysngupta**

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -176,3 +180,53 @@ spec:
           serviceName: frontend
           servicePort: 80
 ```
+
+## Adding Health Probes to your service
+By default, Ingress controller will provision a HTTP GET probe for the exposed pods.
```
Comment (draychev) [inline docs/tutorial.md]: `provision an HTTP GET`

File: docs/tutorial.md [hunk 2]
```diff
@@ -176,3 +180,53 @@ spec:
...
+            path: /
+            port: 80
+          periodSeconds: 3
+          timeoutSeconds: 1
+```
+
+*Note:*
+1) `readinessProbe` and `livenessProbe` are supported when configured with `httpGet`.
+2) Probing on a port other than the one exposed on the pod is currently not supported.
+3) `HttpHeaders`, `InitialDelaySeconds`, `SuccessThreshold` are not supported.
```
Comment (draychev) [inline docs/tutorial.md]: Thanks for thoroughly explaining what is what is not supported!

File: docs/tutorial.md [hunk 3]
```diff
@@ -176,3 +180,53 @@ spec:
...
+
+###  Without `readinessProbe` or `livenessProbe`
+If the above probes are not provided, then Ingress Controller make an assumption that the service is reachable on `Path` specified for `backend-path-prefix` annotation or the `path` specified in the `ingress` definition for the service.
```
Comment (draychev) [inline docs/tutorial.md]: Eventually we might want to add a more thorough description of what the implications of the default probe here are.

File: docs/tutorial.md [hunk 4]
```diff
@@ -176,3 +180,53 @@ spec:
           serviceName: frontend
           servicePort: 80
 ```
+
+## Adding Health Probes to your service
+By default, Ingress controller will provision a HTTP GET probe for the exposed pods.  
+The probe properties can customized by adding a Readiness or a Liveness Probe to your deployment/pod spec.
```
Comment (asridharan) [inline docs/tutorial.md]: The probe properties can be customized by adding a [Readiness] or a [Liveness] probe to your deployment/pod spec.

Could you add reference to the Readiness and Liveness probe documentation.

**Pull request merged**